x/budget/types/budget.go

```diff
@@ -62,10 +55,12 @@ func (budget Budget) Validate() error {
 	return nil
 }
 
+// Expired validates the budget's end time expiration.
 func (budget Budget) Expired(blockTime time.Time) bool {
 	return !budget.EndTime.After(blockTime)
 }
 
+// ValidityAddr validates the bech32 address format.
```

Comment:
Can we rename `ValidityAddr` to `ValidateAddr` or `ValidateAddress`?

Reply:
Actually, this function is identical to `sdk.AccAddressFromBech32`, so we can safely replace `ValidityAddr` with `sdk.AccAddressFromBech32`.

Reply:
Either proposal is fine with me. Renaming to `ValidateAddr` or `ValidateAddress` will make it consistent with `ValidateName`.

Reply:
As we discussed offline, we decided to go remove `ValidityAddr` and replace it with using pure `sdk.AccAddressFromBech32`.